lib/bosh/workspace/helpers/stemcell_helper.rb

```diff
@@ -36,10 +39,38 @@ def project_deployment_stemcells
       end
     end
 
+    def download_stemcell_from_bosh_io(stemcell)
+
+      url=sprintf("https://bosh.io/d/stemcells/%s?v=%s", stemcell.name, stemcell.version)
+
+      response = HTTPClient.new.head(url)
```

Comment:
What if stemcell file is already downloaded? I seems like it is going to refetch stemcell every `bosh deploy` run.